composer.json

```diff
@@ -12,8 +12,7 @@
     ],
     "require": {
         "php": "^7.3",
-        "phpactor/container": "^1.3.3",
-        "phpactor/completion-extension": "~0.2",
+        "phpactor/completion": "^0..4.3",
```

Comment:
`completion-extension` is still [required](https://github.com/phpactor/completion-worse-extension/blob/master/lib/CompletionWorseExtension.php#L33) ?

Reply:
also should be `0.4.3` :)

Reply:
Thanks for fixing the typo :) 
And I don't know for the `completion-extension` maybe we need it for some constants...
I'm a bit lost between the `completion`, `completion-extension` and `completion-worse-extension` ><

Reply:
`completion-extension` is the extension point, `completion` has both the model AND the worse implementations, and `completion-worse-extension` has the integration for the worse implementations.

Finally it would probably make sense to merge them all together.

Reply:
It would be pragmatic and acceptable yes.
It's particularly strange to have the worse implementation in the `completion` package but having a `completion-worse-extenion` to register the implementation from the `completion` package into the container configured in `completion-extension` :D